pkg/storage/interfaces.go

```diff
@@ -112,13 +112,17 @@ type Interface interface {
 	// resourceVersion may be used to specify what version to begin watching,
 	// which should be the current resourceVersion, and no longer rv+1
 	// (e.g. reconnecting without missing any updates).
+	// If resource version is "0", this interface will get current object at given key
+	// and send it in an "ADDED" event, before watch starts.
```

Comment:
If it's not a regression, I'm hesitant to introduce a behavior change to 1.5 this late. I'd be interested to see the adjustment in a PR post-freeze though